test/passing/infix_bind.ml

```diff
@@ -181,3 +181,28 @@ let foo =
       let batch = {repo; tree= old_tree; origin} in
       let b = Batch batch in
       foo
+
+let _ = foo >>= function[@warning "-4"] A -> false | B -> true
+
+let _ =
+  foo >>= function[@warning "-4"]
+  | Afoooooooooooooooooo fooooooooo -> false
+  | Bfoooooooooooooooooooooo fooooooooo -> true
+
+let _ =
+  foo >>= fun [@warning "-4"] x ->
```

Comment:
Is there a reason to add a space before the attribute on `fun` but not on `function` ?